configs/centernet/centernet_resnet18_dcnv2_140e_coco.py

```diff
@@ -0,0 +1,100 @@
+_base_ = [
+    '../_base_/datasets/coco_detection.py',
+    '../_base_/schedules/schedule_1x.py', '../_base_/default_runtime.py'
```

Comment:
The original centernet uses Adam but here we use SGD. It would be better to show the optimizer setting in this config instead of hiding it in schedule_1x.py. Moreover, since the lr config and runner config are also changed, is it really necessary to inherit the schedule_1x config?

Reply:
I plan to detail the changes compared to the original version in the README and It's okay not to inherit schedule_1x.